saleor/payment/gateways/stripe/webhooks.py

```diff
@@ -191,9 +200,11 @@ def _finalize_checkout(
     )
 
     try:
-        # when checkout total value is different than total amount from payments
-        # it means that some products has been removed during the payment was completed
-        if checkout_total.gross.amount != payment.total:
+        # when checkout total amount is less than total amount from payments
+        # it means that something changed in the checkout and we make a refund
+        # if the checkout is overpaid we allow to create the order and handle it
+        # by staff.
+        if checkout_total.gross.amount > payment.total:
             payment_refund_or_void(payment, manager, checkout_info.channel.slug)
             raise ValidationError(
                 "Cannot complete checkout - some products do not exist anymore."
```

Comment:
is this error message valid now? shouldn't it say that checkout total was changed and payment doesn't cover it anymore?

Reply:
Good catch @jakubkuc , thanks.